website/docs/guides/1.15.0-upgrade-guide.html.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas Provider 1.15.0: Upgrade and Information Guide"
+sidebar_current: "docs-mongodbatlas-guides-1150-upgrade-guide"
+description: |-
+MongoDB Atlas Provider 1.15.0: Upgrade and Information Guide
+---
+
+# MongoDB Atlas Provider 1.15.0: Upgrade and Information Guide
+
+The Terraform MongoDB Atlas Provider version 1.15.0 has a number of new and exciting features.
+
+**New Resources, Data Sources, and Features:**
```

Comment:
do we keep sections when empty?

Reply:
Since this is an initial version of the file and when we do the actual release this section might include information, I left it